pkg/util/converter/prom_test.go

```diff
@@ -16,9 +16,6 @@ import (
 const update = true
 
 func TestReadPromFrames(t *testing.T) {
-	// FIXME:
-	// skipping test due to flaky behavior
-	t.Skip()
```

Comment:
What made them flaky before that is no longer flaky?

Reply:
good question. i do not know if they are still flaky or not :-( ... i'm operating on the assumption that having these maybe-flaky and running is better right now than having them not-running at all.

the original PR that disabled the tests https://github.com/grafana/grafana/pull/52243 has some info about the test-failure, but it is not failing for me.

Reply:
Works for me as well:

```
/usr/local/go/bin/go test -timeout 120s -tags integration -run ^TestReadPromFrames$ github.com/grafana/grafana/pkg/util/converter -count=100 
ok      github.com/grafana/grafana/pkg/util/converter   30.460s
```